src/styles/utilities/variables.scss

```diff
@@ -0,0 +1,18 @@
+$calendar-border-vert: 20px;
+$calendar-border-hor: 10px;
+$calendar-gaps: 1px;
+$day-size: 100px;
+$day-transform-time: 0.5s;
+$day-background: #eee;
+$day-background-hover: pink;
+$day-border-color: #000;
+$day-font: arial, helvetica, sans-serif;
```

Comment:
```suggestion
$day-font: Arial, Helvetica, sans-serif;
```

Reply:
I vaguely recall linter being upset with me when fonts where written in upper case, which prompted me to make them like this in the first place.